### PR TITLE
Added tests for 'oscap xccdf generate guide' with --benchmark-id option.

### PR DESCRIPTION
**Pull request by rsprudencio**

### CODE REVIEW

File: tests/API/XCCDF/guide/Makefile.am [hunk 1]
```diff
@@ -11,4 +11,5 @@ EXTRA_DIST = \
 	all.sh \
 	content-xccdf11.xml \
 	content-xccdf12.xml \
-	content-value-xccdf11.xml
+	content-value-xccdf11.xml \
+	benchmark_id_test-ds.xml
```
Comment (yuumasato) [inline tests/API/XCCDF/guide/Makefile.am]: `benchmark_id_test-ds.xml` was not added to this commit, and caused the jenkins fail

Comment (rsprudencio) [inline tests/API/XCCDF/guide/Makefile.am]: I feel so stupid, I was wondering why it was working in my machine :rofl:

File: tests/API/XCCDF/guide/benchmark_id_test-ds.xml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_centos7:def:1</check>
+      </cpe-item>
+      <cpe-item name="cpe:/o:scientificlinux:scientificlinux:7">
+            <title xml:lang="en-us">Scientific Linux 7</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="ssg-rhel7-cpe-oval.xml">oval:ssg-installed_OS_is_sl7:def:1</check>
+      </cpe-item>
+    </cpe-list>
+  </ds:component>
```
Comment (mpreisler) [inline tests/API/XCCDF/guide/benchmark_id_test-ds.xml]: The CPE component is not used, can you please remove it? It's not necessary for testing purposes.

**Comment by mpreisler** [comment]
I just have one minor nitpick about the CPE component, otherwise this looks good.

The OVAL and XCCDF could be more minimal but that's not a big deal.

**Comment by mpreisler** [comment]
I decided to merge this, what I pointed out is an issue but is not really important.

**Pull request merged**